### PR TITLE
Usdc bridge deployment

### PR DESCRIPTION
**Pull request by gvladika**
Deployment and set-up process for bridged USDC standard.

### CODE REVIEW

File: scripts/usdc-bridge-deployment/README.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+
+Circle’s Bridged USDC Standard is a specification and process for deploying a bridged form of USDC on EVM blockchains with optionality for Circle to seamlessly upgrade to native issuance in the future.
+
+We provide custom USDC gateway implementation (for parent and child chain) that follows Bridged USDC Standard. These contracts can be used by new Orbit chains. This solution will NOT be used in existing Arbitrum chains. On parent chain contract `L1USDCGateway` is used in case child chain uses ETH as native currency, or `L1OrbitUSDCGateway` in case child chain uses custom fee token. On child chain `L2USDCGateway` is used. For the USDC token contracts, Circle's referent implementation is used.
+
+This doc describes how to deploy USDC bridge compatible with both Arbitrum's token bridge and Circle’s Bridged USDC Standard.Also steps for transition to native USDC issuance are provided.
+
+## Assumptions
+
+It is assumed there is already USDC token deployed and used on the parent chain. If not, follow the instructions in the Circle's `stablecoin-evm` repo to deploy one.
```
Comment (gzeoneth) [inline scripts/usdc-bridge-deployment/README.md:13]: I think only Circle should deploy it on the parent chain, if it is not deployed by circle this standard is not applicable

File: scripts/usdc-bridge-deployment/README.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+
+## Transition to native USDC
+
+Once transition to native USDC is agreed on, following steps are required:
+
+- L1 gateway owner pauses deposits on parent chain by calling `pauseDeposits()`
+- L2 gateway owner pauses withdrawals on child chain by calling `pauseWithdrawals()`
+- master minter removes the minter role from the child chain gateway
+  - there should be no in-flight deposits when minter role is revoked. If there are any, they should be executed (can be done by anyone by claiming the failed retryable ticket which does the USDC depositing)
+  - if minter role is revoked before in-flight deposits are claimed, those funds can’t be minted. One option is to leave the gateway’s minter role, but decrease the allowance to match the total amount of unclaimed deposits
```
Comment (gzeoneth) [inline scripts/usdc-bridge-deployment/README.md:74]: remove

**Review by gzeoneth — CHANGES_REQUESTED** [review]

File: scripts/usdc-bridge-deployment/README.md (hunk 1 above)
Comment (gvladika) [reply scripts/usdc-bridge-deployment/README.md]: Right https://github.com/OffchainLabs/token-bridge-contracts/pull/93/commits/5065de4e9a73d4958dff8b432751f5c6c925edab

File: scripts/usdc-bridge-deployment/README.md (hunk 2 above)
Comment (gvladika) [reply scripts/usdc-bridge-deployment/README.md]: https://github.com/OffchainLabs/token-bridge-contracts/pull/93/commits/5065de4e9a73d4958dff8b432751f5c6c925edab

**Review by gzeoneth — APPROVED** [review]
LGTM

**Pull request merged**